scripts/cfengine-build-host-setup.cf

```diff
@@ -0,0 +1,199 @@
+body file control
+{
+  inputs => { "$(sys.libdir)/stdlib.cf" };
+}
+# assumes that the operating system has working repo lists and has been updated/upgraded already
+
+bundle agent cfengine_build_host_setup
+{
+  packages:
+    debian_9|debian_10::
+      "python-psycopg2";
```

Comment:
Attribute-less packages promise, so it's going to hit the generic package method